data/en/qpses160_0002.json

```diff
@@ -291,7 +291,7 @@
                                         "list": [
                                             "Agency workers paid directly from the agency payroll",
                                             "Self-employed workers",
-                                            "Voluntary Workers",
+                                            "Voluntary workers",
```

Comment:
This change is being made on a separate PR so can probably be removed from this branch.